package.json

```diff
@@ -80,6 +80,7 @@
     "html-webpack-plugin": "^3.2.0",
     "husky": "^1.1.2",
     "jest": "^22.4.3",
+    "jsdom": "^13.0.0",
```

Comment:
don't thinnk this is technically required since we are running jest with `jsdom` option, however linting moans about it